src/tseda/vpages/ignn.py

```diff
@@ -251,9 +362,19 @@ def gnn(self):
         return df
 
     @pn.depends("sorting", "sort_order")
-    def __panel__(self):
+    def __panel__(self) -> Union[pn.pane.plot.Bokeh, pn.pane.Alert, Any]:
```

Comment:
I was unable to make the actual `pn.panel` format work, so I put `Any` here for now. Considering there was previously no check at all I think it is fine, but preferably it should be updated